src/app/shared/util/date/date.format.ts

```diff
@@ -20,6 +20,14 @@ export const monthDateYear = (date: Date | moment.Moment, separator = true): str
   }
 };
 
+export const monthDateYearLocalTZ = (date: Date | moment.Moment, separator = true): string => {
```

Comment:
Still a little confusing that the hourly graph shows UTC times.  But hey, getting better!

Reply:
It's actually showing local timezone with hourly data in the chart and in the data for export, or did I miss that somewhere? The date selectors are still UTC based when hourly is selected. It is a weird mix.